crates/stdarch-gen/Makefile

```diff
@@ -0,0 +1,4 @@
+all:
```

Comment:
I'd rather not use makefiles. `cargo run` should generate the files, and `cargo test` should verify that the files in the other repos are uptodate.

Reply:
That's a really nice idea! I didn't think of that!